frontend/src/services/http/http.service.ts

```diff
@@ -20,10 +41,17 @@ class Http {
       .catch(this.throwError);
   }
 
-  private getHeaders(): Headers {
+  private getHeaders(contentType?: ContentType, hasAuth?: boolean): Headers {
     const headers = new Headers();
 
-    headers.append(HttpHeader.CONTENT_TYPE, 'application/json');
+    if (contentType) {
```

Comment:
You has removed content-type application/json by default, so you need to pass it in frontend service for sign-up and sign-in, because they won't work without it

Reply:
Ok, but i thought it's already done here 
![image](https://user-images.githubusercontent.com/35830708/183725426-ce945e36-89ec-443e-81f4-d400dcadc523.png)

Reply:
Ok, than add hasAuth: false to them